bin/puppet_install.sh

```diff
@@ -40,85 +44,80 @@ fi
 }
 
 setup_fedora() {
   release=$1
   echo_title "Uninstalling existing Puppet"
-  $sudo_command yum erase -y puppet-agent puppet puppetlabs-release puppetlabs-release-pc1 >/dev/null 2>&1
+  $sudo_command yum erase -y puppet-agent puppet puppetlabs-release puppetlabs-release-pc1 &>/dev/null
 
   echo_title "Adding repo for Puppet"
-  $sudo_command rpm -ivh https://yum.puppetlabs.com/puppet$puppet_version-release-fedora-${release}.noarch.rpm
+  $sudo_command yum install -y https://yum.puppetlabs.com/puppet$puppet_version-release-fedora-${release}.noarch.rpm &>/dev/null
 
   sleep 2
   echo_title "Installing Puppet"
-  $sudo_command yum install -y puppet-agent
+  $sudo_command yum install -y puppet-agent &>/dev/null
 }
 
 setup_amazon() {
   echo_title "Uninstalling existing Puppet"
-  $sudo_command yum erase -y puppet-agent puppet puppetlabs-release puppetlabs-release-pc1 >/dev/null 2>&1
+  $sudo_command yum erase -y puppet-agent puppet puppetlabs-release puppetlabs-release-pc1 &>/dev/null
 
-  echo_title "Adding repo for Puppet 4"
-  $sudo_command rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-6.noarch.rpm >/dev/null 2>&1
-  $sudo_command yum-config-manager --enable epel
-  $sudo_command yum-config-manager --setopt="puppetlabs-pc1.priority=1" --save
+  echo_title "Adding repo for Puppet"
+  $sudo_command yum install -y https://yum.puppetlabs.com/puppet$puppet_version-release-el-${release}.noarch.rpm &>/dev/null
 
   sleep 2
   echo_title "Installing Puppet"
-  $sudo_command yum install -y puppet-agent >/dev/null 2>&1
+  $sudo_command yum install -y puppet-agent &>/dev/null
 }
 
 setup_suse() {
   echo_title "Uninstalling existing Puppet"
-  $sudo_command zypper remove -y puppet >/dev/null 2>&1
-  $sudo_command zypper remove -y puppet-agent >/dev/null 2>&1
-  $sudo_command zypper remove -y puppetlabs-release >/dev/null 2>&1
-  $sudo_command zypper remove -y puppetlabs-release-pc1 >/dev/null 2>&1
+  $sudo_command zypper remove -y puppet puppet-agent puppetlabs-release puppetlabs-release-pc1 &>/dev/null
 
   echo_title "Adding repo for Puppet"
-  $sudo_command wget https://yum.puppetlabs.com/puppet/puppet-release-sles-$1.noarch.rpm 2>&1
-  $sudo_command rpm -ivh puppet-release-sles-$1.noarch.rpm 2>&1
+  $sudo_command zypper --no-gpg-checks --non-interactive install https://yum.puppetlabs.com/puppet$puppet_version-release-sles-${release}.noarch.rpm &>/dev/null
 
   sleep 2
   echo_title "Installing Puppet"
-  $sudo_command zypper --no-gpg-checks  --non-interactive install puppet-agent
+  $sudo_command zypper --no-gpg-checks --non-interactive install puppet-agent &>/dev/null
 }
 
 setup_apt() {
   case $1 in
     3*) codename=cumulus ;;
-    8) codename=jessie  ;;
+    8) codename=jessie   ;;
     9) codename=stretch  ;;
-    10) codename=buster ;;
-    14.04) codename=trusty  ;;
+    10) codename=buster  ;;
+    14.04) codename=trusty ;;
     16.04) codename=xenial ;;
     18.04) codename=bionic ;;
-    18.10) codename=bionic ;;
```

Comment:
why are these removed? Should work the same with the packages packed for lts releases

Reply:
i thought because its EOL….